champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/knight/sword/DefensiveStance.java

```diff
@@ -113,23 +113,27 @@ public void onDamage(CustomDamageEvent event) {
         Gamer gamer = championsManager.getClientManager().search().online(player).getGamer();
         if (!gamer.isHoldingRightClick()) return;
 
+
         int level = getLevel(player);
         if (level > 0) {
+            event.setDamage(event.getDamage() * (1.0 - getDamageReduction(level)));
```

Comment:
You are reducing the damage even if the player is hit from behind, keep this where it was